akka-diagnostics/src/test/scala/akka/diagnostics/ConfigCheckerSpec.scala

```diff
@@ -865,45 +865,6 @@ class ConfigCheckerSpec extends AkkaSpec {
       assertDisabled(c, "remote-prefer-cluster")
     }
 
-    "not warn about the dynamic hostnames when artery is used" in {
```

Comment:
note that something is fishy here, the test title says "not warn", but it's actually asserting that there is a warning